<launch>

    <node pkg="example_service" name="fibonacci_service" type="example_service_node.py"/>
    <node pkg="example_action_server" name="fibonacci" type="example_action_server_node.py"/>
    <node pkg="homework4" name="homework4_part2" type="homework4_part2.py"/>

</launch>